pallets/api/src/fungibles/tests.rs

```diff
@@ -272,14 +272,16 @@ fn decrease_allowance_works() {
 				BadOrigin.with_weight(WeightInfo::approve(0, 0))
 			);
 		}
+		assets::create_mint_and_approve(owner, token, owner, value, spender, value);
+		assert_eq!(Assets::allowance(token, &owner, &spender), value);
```

Comment:
Testing something again that doesn't need to be.

Reply:
This is what we have originally, I just reorder.